.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Hi! Before opening a new issue, please make sure it has not already been
+reported. In addition, please ask on Slack if the latest code has been deployed.
+It's possible that the bug you are seeing is fixed in the code, but hasn't been
+deployed yet.
+
+Once you are sure the issue is valid, please fill out the details below.
+Please try to be as specific as possible when describing the steps to reproduce.
+
+Thanks!
+
+### URL where issue was found
+
+
+### Date and time when issue was found (if relevant)
```

Comment:
It may be worth mentioning that a specific time is helpful like '4:54PM EDT', not 'around 4pm'.